bedrock/mozorg/templates/mozorg/developer/includes/sub-nav.html

```diff
@@ -7,7 +7,26 @@
 {% block sub_nav_class %}technology{% endblock %}
 
 {% block sub_nav_primary_links %}
-<li><a href="{{ url('mozorg.technology') }}" data-link-name="Web Innovations" data-link-type="nav" data-link-position="subnav">{{ _('Web Innovations') }}</a></li>
-<li><a href="{{ url('mozorg.developer') }}" data-link-name="Mozilla Developers" data-link-type="nav" data-link-position="subnav">{{ _('Mozilla Developers') }}</a></li>
-<li><a href="https://developer.mozilla.org" data-link-name="MDN" data-link-type="nav" data-link-position="subnav">{{ _('MDN') }}</a></li>
+<li>
+  {% if 'technology' in request.path %}
+    <a href="{{ url('mozorg.developer') }}" data-link-name="Developer Hub" data-link-type="nav" data-link-position="subnav">{{ _('Developer Hub') }}</a>
+  {% else %}
+    <a href="{{ url('mozorg.technology') }}" data-link-name="Technology Hub" data-link-type="nav" data-link-position="subnav">{{ _('Technology') }}</a>
+  {% endif %}
+</li>
+<li>
+  <a href="https://research.mozilla.org/" data-link-name="Research" data-link-type="nav" data-link-position="subnav">{{ _('Research') }}</a>
```

Comment:
I do wonder if we should also have some utm params for these external links in the secondary nav? (the "support" link to SUMO in the Firefox sub nav has some as an example).

Reply:
And they also not have in the drawer, perhaps I should use the same here? but perhaps keep the `data-link-position="subnav"` here?

Reply:
Both the sub nav and the drawer have specific data attributes, so I would just apply the same ones that you see in existing examples.